site/en/tutorials/load_data/tf-records.ipynb

```diff
@@ -496,7 +684,35 @@
       "cell_type": "code",
       "source": [
         "filenames = ['test.tfrecords']\n",
-        "dataset = tf.data.TFRecordDataset(filenames)"
+        "dataset_raw = tf.data.TFRecordDataset(filenames)\n",
+        "\n",
+        "dataset_raw"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "6_EQ9i2E_-Fz",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "At this point the dataset contains serialized `tf.train.Example` Messages. When iterated over it returns these as scalar string tensors:\n",
+        "\n",
+        "Note: iterating over a `tf.data.Dataset` only works with eager execution enabled."
+      ]
+    },
+    {
+      "metadata": {
+        "id": "hxVXpLz_AJlm",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "for record in dataset_raw.take(10):\n",
```

Comment:
consider calling the loop variable `raw_record` and the one used for the parsed dataset `parsed_record`